app/src/main/java/io/homeassistant/companion/android/notifications/MessagingManager.kt

```diff
@@ -136,6 +136,7 @@ class MessagingManager @Inject constructor(
         const val HIGH_ACCURACY_UPDATE_INTERVAL = "high_accuracy_update_interval"
         const val PACKAGE_NAME = "package_name"
         const val CONFIRMATION = "confirmation"
+        const val RELATIVE_VOLUME = "relative_volume"
```

Comment:
Which parameter name do you think is the best?
- `relative_volume`
- `relative`
- `adjust`
- `delta`